pkg/clusters/addons/kong/kong.go

```diff
@@ -296,3 +360,75 @@ func urlForService(ctx context.Context, cluster clusters.Cluster, nsn types.Name
 
 	return nil, fmt.Errorf("service %s has not yet been provisoned", service.Name)
 }
+
+func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name string) error {
+	license := os.Getenv("KONG_ENTERPRISE_LICENSE")
+	if license == "" {
+		return fmt.Errorf("failed kong enterprise license from environment setting")
+	}
```

Comment:
As I mentioned in several previous comments we shouldn't be requiring environment variables for arguments to our code, this should be a function argument:

```suggestion
func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name, license string) error {
```

Then callers like the integration tests, KIC, or the `ktf` CLI can pull the license from an ENV at a higher level, but all arguments to the kong addon configuration are consistently parameters.

Reply:
moved ENV to higher level https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-3d68f23e86db256aa769ae3f812e898e238f059fd95d96ff32a452981f588070R184